components/likers/modal.ts

```diff
@@ -32,6 +32,12 @@ export default class AppLikesModal extends BaseModal {
 	currentPage = 0;
 	users: User[] = [];
 
+	// Just for display purposes, if we have more users than the count passed in, display that instead.
+	// This can happen when the count was fetched before new users were added to the list.
+	get realCount() {
```

Comment:
Is the only time this happens when you click into the modal and it loads more users, or when you paginate through the likers?

Reply:
When you load eg. a post view page, it loads the total amount of likes from backend. This number is at that time for example 4. Then you leave the page open until someone else likes the post, then you open the modal. This modal will now fetch the users that liked the post, which will be 5 users, but since we pass in 4 as a count, the title of that modal will display "4 likers".
This makes sure that if that scenario happens, we always show the max number in the title, whether that be the count we pass in or the number of users actually displayed.